fintech-examples/fintech-ui/src/index.html

```diff
@@ -5,7 +5,7 @@
     <title>Fintech UI</title>
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="icon" type="image/x-icon" href="favicon.ico" />
+    <link rel="icon" type="image/x-icon" href="./assets/icons/card_payment.png" />
```

Comment:
This icon should have clear name `favicon` in the resources as it would not be used anywhere else. You need to create new asset